libs/components/popovers/src/lib/modules/dropdown/dropdown.component.spec.ts

```diff
@@ -123,6 +124,12 @@ describe('Dropdown component', function () {
           provide: SkyThemeService,
           useValue: mockThemeService,
         },
+        {
+          provide: SKY_STACKING_CONTEXT,
+          useValue: {
+            zIndex: new BehaviorSubject(111),
```

Comment:
What do you think about passing in `asObservable` here, and then completing the `BehaviorSubject` in the `afterEach`? I'd imagine the tests might get clogged up with so many uncompleted observables floating around. 🤔